storage/rocksdb/rdb_cf_options.h

```diff
@@ -46,7 +46,8 @@ class Rdb_cf_options {
   Rdb_cf_options &operator=(const Rdb_cf_options &) = delete;
   Rdb_cf_options() = default;
 
-  void get(const std::string &cf_name,
+  /* bool true return indicates cf_name was found */
+  bool get(const std::string &cf_name,
```

Comment:
Consider MY_NODISCARD annotation to catch unintended return value non-checks easier

Reply:
That's a good consideration, but I think not all calls to this will care if the options returned were from a specific column family override or from the 'default' set.  I traced the calls back and also went ahead and tried it and it seems that all current uses do check the result so it can't hurt to make it more strict and then loosen it up again later on if need be.